### PR TITLE
Add ObituaryDesign

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?
Add a new `Design`, `ObituaryDesign`, which was missing.

### CODE REVIEW

File: client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala [hunk 1]
```diff
@@ -84,6 +84,7 @@ object CapiModelEnrichment {
         tagExistsWithId("tone/analysis") -> AnalysisDesign,
         tagExistsWithId("tone/comment") -> CommentDesign,
         tagExistsWithId("tone/letters") -> LetterDesign,
+        tagExistsWithId("tone/obituaries") -> ObituaryDesign,
```
Comment (buck06191) [inline client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala:87]: I'd put this above stuff like comment  and analysis - I can't think of a situation where we'd want something to come through as a different category to Obituary, so we probably want that near the top

**Review by buck06191 — CHANGES_REQUESTED** [review]
I think the obituary needs moving up the predicates list

File: client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala (hunk 1 above)
Comment (shtukas) [reply client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala]: Done

**Review by buck06191 — APPROVED** [review]
LGTM

**Pull request merged**